Source/WebKitLegacy/mac/WebView/WebUIDelegatePrivate.h

```diff
@@ -144,7 +144,6 @@ extern NSString *WebConsoleMessageJSMessageSource;
 extern NSString *WebConsoleMessageNetworkMessageSource;
 extern NSString *WebConsoleMessageConsoleAPIMessageSource;
 extern NSString *WebConsoleMessageStorageMessageSource;
-extern NSString *WebConsoleMessageAppCacheMessageSource;
```

Comment:
we may need to keep this if there's a client elsewhere at Apple that still uses it